src/coreComponents/linearAlgebra/unitTests/testExternalSolvers.cpp

```diff
@@ -83,7 +83,7 @@ LinearSolverParameters params_CG_AMG()
   parameters.isSymmetric = true;
   parameters.solverType = LinearSolverParameters::SolverType::cg;
   parameters.preconditionerType = LinearSolverParameters::PreconditionerType::amg;
-  parameters.amg.smootherType = geos::LinearSolverParameters::AMG::SmootherType::fgs;
+  parameters.amg.smootherType = geos::LinearSolverParameters::AMG::SmootherType::sgs;
```

Comment:
@victorapm @klevzoff: do you agree with this change? With `fgs` as a smoother, AMG preconditioned CG does not converge (not surprisingly) in 300 iterations to the required tolerance (1e-8) for the 2D Laplace unit test (problem size 10,000). With a symmetric smoother like `sgs`, CG converges in 6 iterations, as one would expect.

Reply:
Yes, that makes much more sense to me